### PR TITLE
Improve mock backend logging

### PR DESCRIPTION
**Pull request by llowinge**
This PR is follow up to @ppalaga work in https://github.com/apache/camel-quarkus/pull/1782.

### CODE REVIEW

**Comment by llowinge** [comment]
Currently blocked by https://github.com/quarkusio/quarkus/issues/12250

File: integration-tests/telegram/src/main/java/org/apache/camel/quarkus/component/telegram/it/TelegramRoutes.java [hunk 1]
```diff
@@ -21,37 +21,80 @@
...
+        result.setCamelContext(getContext());
+        result.setBaseUri(getBaseUri());
+        result.setAuthorizationToken(authToken);
+        return result;
+    }
+
     @Override
     public void configure() throws Exception {
+        if (MockBackendUtils.startMockBackend(true)) {
+            /* Start the mock Telegram API unless the user did export CAMEL_QUARKUS_FALLBACK_MOCK=false */
```
Comment (llowinge) [inline integration-tests/telegram/src/main/java/org/apache/camel/quarkus/component/telegram/it/TelegramRoutes.java:76]: @ppalaga This is a typo ? `CAMEL_QUARKUS_FALLBACK_MOCK` ?

Comment (ppalaga) [reply integration-tests/telegram/src/main/java/org/apache/camel/quarkus/component/telegram/it/TelegramRoutes.java:76]: I do not see an issue. What would you expect there?

Comment (ppalaga) [reply integration-tests/telegram/src/main/java/org/apache/camel/quarkus/component/telegram/it/TelegramRoutes.java:76]: Oh, I see, it should be `CAMEL_QUARKUS_START_MOCK_BACKEND=false`

Comment (ppalaga) [reply integration-tests/telegram/src/main/java/org/apache/camel/quarkus/component/telegram/it/TelegramRoutes.java:76]: Thanks, good catch!

**Pull request merged**